internal/db/issue.go

```diff
@@ -407,14 +407,14 @@ func (issue *Issue) ReadBy(uid int64) error {
 	return UpdateIssueUserByRead(uid, issue.ID)
 }
 
-func updateIssueCols(e Engine, issue *Issue, cols ...string) error {
-	_, err := e.ID(issue.ID).Cols(cols...).Update(issue)
+func updateIssueCols(e Engine, issue *Issue) error {
```

Comment:
I think the function should be renamed as well
```suggestion
func updateIssue(e Engine, issue *Issue) error {
```